cli/flox-rust-sdk/src/models/pkgdb.rs

```diff
@@ -80,30 +83,29 @@ impl<'de> Deserialize<'de> for PkgDbError {
     where
```

Comment:
@zmitchell ?

Reply:
There was a reason at the time, but I don't remember what it was (probably should have been a comment, my bad). Maybe it was so that we could control the error messages better so the user doesn't see the raw `serde_json` error if it can't parse it?

Reply:
Seems like maybe we can just drop the custom deserializer then?

Reply:
It's a tradeoff, the code will be simpler, but the error messages will be worse in the case that the JSON can't be parsed. I'm fine with whatever you decide, but we'll likely end up doing something similar the first time a user sees a `serde_json` error.

Reply:
Just commenting for now